CST-to-AST/generic-functions.lisp

```diff
@@ -96,3 +211,35 @@
   (:method (client ast ctype context origin environment)
     (declare (ignore client ctype context origin environment))
     ast))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;
+;;; Generic function ATTRIBUTES.
+;;;
+;;; FIXME(paul) Add doc?
+
+(defgeneric attributes (client description environment))
+
+(defmethod attributes
+    (client (description trucler:authentic-function-description) environment)
+  (declare (ignore client description environment))
+  (cleavir-attributes:default-attributes))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;
+;;; Generic function POLICY.
+;;;
+;;; FIXME(paul) Add doc?
```

Comment:
The documentation should include pretty extensive information here, including probably an example of including policy information in an environment so that it does not have to be repeatedly computed. I suspect that the default recomputation will be pretty bad for performance.